doc/layouts/shortcodes/cli-only.html

```diff
@@ -0,0 +1,4 @@
+<div class="panel panel-primary panel-header">
+  <h3>Command-line interface only</h3>
+  The instructions on this page use the command-line interface (CLI). <a href="{{- ref . "/gettingstarted/cli" -}}">Learn how to get started with the CLI</a>
```

Comment:
Shouldn't we consistentl use ref or relref everywhere?

Reply:
They have their purpose; `relref` within sections, `ref` cross sections. That allows for promoting/demoting sections by only updating entry links. I'm not in favor of `../../../../gettingstarted` (ok, bit exaggerated)